templates/terraform/processing-project/variables.tf

```
@@ -37,3 +37,13 @@ variable kubeconfig_path {
   type = string
   description = "Local path where kubeconfig for the processing GKE cluster should be written."
 }
+
+variable access_emails {
+  type = list(string)
+  description = "Emails to share bucket read access with."
+}
+
+variable deletion_age {
```

```suggestion
variable deletion_age_days {
```
To make the unit super clear